### ISSUE TITLE
add argument to delete project by their ID

### ISSUE THREAD

**Issue by farheinheigt** [Syslifters/reptor#190]
It is not possible to delete a project by its ID.
When you have at least two projects with identical names, you can't specify the right one by its ID.
you can delete all projects with the same name, or delete the good project on the web interface.

**Comment by aronmolnar** [comment]
This is true. It should be quiet easy to add a `--project-id` argument to [DeleteProject.py](https://github.com/Syslifters/reptor/blob/main/reptor/plugins/projects/DeleteProjects/DeleteProjects.py) and then reference the project by id.

Do you think you can make the happen and create a pull request?
